assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -218,7 +218,7 @@ export function LeftRail() {
       </div>
       <Content>
         <AppInfo>
-          {__('\u00A9 2020 Google', 'web-stories')}
+          {sprintf('\u00A9 %s Google', new Date().getFullYear())}
```

Comment:
Is the translators comment needed here as well?

Reply:
No, as this is not translated anymore (I removed `__()`).

There's no point in making this translatable really.

Reply:
Alright, that's universal in all languages?

Reply:
Are there no other countries out there our they own characters for years.

Reply:
Seems like no. Here are 3 examples of languages that use these numbers and are not even the same alphabet.

https://zeenews.india.com/hindi/india/modi-cabinet-meeting-starts-msp-likely-to-be-announced-for-rabi-crops/751728
https://hebrewnews.com/article/35904
https://www.bbc.com/arabic/science-and-tech-54234237

Side note, no idea what these articles say, please don't judge me if they say something bad.